projects/sturdy/index.js

```diff
@@ -24,27 +27,37 @@ const priceInUSD = (value, decimals, price) => {
     return (value / Math.pow(10, decimals) * price / Math.pow(10, 8)).toFixed(2);
 }
 
-async function fetch(borrow) {
+async function fetch(borrow, chain) {
+    const graphUrl = chain == 'fantom' ? graphUrl_ftm : graphUrl_eth;
     const { reserves } = await request(graphUrl, graphQuery);
-    const tvl = reserves.reduce((sum, reserve) => {
+    let tvl = reserves.reduce((sum, reserve) => {
         const value = borrow ? reserve.availableLiquidity : reserve.totalLiquidity;
         return sum + +priceInUSD(value, reserve.decimals, reserve.price.priceInEth);        
     }, 0);
+
+    if (chain == 'ethereum') {
```

Comment:
change this condition to 


    if (chain !== 'ethereum')
        return toUSDTBalances(tvl)

    return {
        ethereum: tvl /1e10
    }
    
  remove dependency on `getTokenPriceCoinGecko`